build.zig

```diff
@@ -102,6 +102,32 @@ pub fn build(b: *std.Build) !void {
         if (want_examples) try buildExamples(b, optimize, target, module);
     }
     if (want_core) {
+        if (Platform.fromTarget(target.result) == .linux) {
+            const lib = b.addStaticLibrary(.{
+                .name = "core-wayland",
+                .target = target,
+                .optimize = optimize,
+            });
+            lib.addCSourceFile(.{
+                .file = b.path("src/core/linux/wayland.c"),
```

Comment:
This is only needed because translate-c can't handle some of the types in the header, right?

Reply:
Filed https://github.com/hexops/mach/issues/1269 as a follow-up issue

Reply:
Pretty sure that's the issue, yeah.